data/src/main/java/com/example/data/api/models/UserCandidateApiModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.data.api.models
+
+data class UserCandidateApiModel(
+    val name: String,
```

Comment:
I'd suggest making all API models parameters optional so we can use default values during mapping.